src/ccalls.jl

```diff
@@ -29,42 +29,42 @@ function CanAssignGlobalVariable(name::String)
              (Ptr{UInt8},), name)
 end
 
-function AssignGlobalVariable(name::String, value::MPtr)
+function AssignGlobalVariable(name::String, value::GapObj)
```

Comment:
Shouldn't this be 
```suggestion
function AssignGlobalVariable(name::String, value::Obj)
```
so that we can also assign bools, FFEs and immediate integers? (That is of course an existing problem, not new in this PR).

Well, and then I guess we should actually test all this variants. And also similarly for a bunch of other functions in this file...

Reply:
Yes, indeed.